python-lib/spacy_tokenizer.py

```diff
@@ -183,6 +183,7 @@ def _create_spacy_tokenizer(self, language: AnyStr) -> Language:
                 os.environ["PYTHAINLP_DATA_DIR"] = mkdtemp()  # dummy temp directory
             if language in SPACY_LANGUAGE_MODELS and self.use_models:
                 nlp = spacy.load(SPACY_LANGUAGE_MODELS[language])
+                nlp.remove_pipe("ner")
```

Comment:
The spacy tokenizer module should be universal to all plugins, hence we should avoid hardcoding stuff that is specific to one task. Here, the right thing to do would be to expose a class instance argument called "nlp_pipe_deactivate" which lets you choose which components to deactivate.

Reply:
Side note: For good separation of purposes, this would be better addressed by a dedicated CH card and PR.

Reply:
closing as I opened a dedicated PR